BDArmory/Weapons/Missiles/MissileBase.cs

```diff
@@ -700,6 +731,24 @@ protected void UpdateRadarTarget()
 
             float angleToTarget = Vector3.Angle(radarTarget.predictedPosition - transform.position, GetForwardTransform());
 
+            if (hasIOG == true)
```

Comment:
The `== true` is redundant here and below, since `if (myVar == true)` is the same as `if (myVar)` for boolean values.